src/test/java/api/PrecedingSucceedingTitlesApiExamples.java

```diff
@@ -0,0 +1,496 @@
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class PrecedingSucceedingTitlesApiExamples extends ApiTests {
+
+  @Test
+  public void canCreateAnInstanceWithUnconnectedPrecedingTitles()
+    throws InterruptedException, MalformedURLException, TimeoutException,
+    ExecutionException {
+
+    String precedingSucceedingTitleId1 = UUID.randomUUID().toString();
+    String precedingSucceedingTitleId2 = UUID.randomUUID().toString();
+    JsonArray precedingTitles = getUnconnectedPrecedingSucceedingTitle(
+      precedingSucceedingTitleId1, precedingSucceedingTitleId2);
+
+    JsonObject smallAngryPlanetJson = smallAngryPlanet(UUID.randomUUID());
+    smallAngryPlanetJson.put("precedingTitles", precedingTitles);
+
+    IndividualResource createdInstance = instancesClient.create(smallAngryPlanetJson);
+
+    JsonArray actualPrecedingTitles = createdInstance.getJson().getJsonArray("precedingTitles");
+    JsonObject actualPrecedingTitle1 = getRecordById(actualPrecedingTitles, precedingSucceedingTitleId1);
+    JsonObject actualPrecedingTitle2 = getRecordById(actualPrecedingTitles, precedingSucceedingTitleId2);
+
+    assertPrecedingSucceedingTitles(actualPrecedingTitle1, precedingTitles.getJsonObject(0), null,
+      createdInstance.getId().toString());
+    assertPrecedingSucceedingTitles(actualPrecedingTitle2, precedingTitles.getJsonObject(1), null,
+      createdInstance.getId().toString());
+  }
+
+  @Test
+  public void canCreateAnInstanceWithUnconnectedSucceedingTitles()
+    throws InterruptedException, MalformedURLException, TimeoutException,
+    ExecutionException {
+
+    String precedingSucceedingTitleId1 = UUID.randomUUID().toString();
+    String precedingSucceedingTitleId2 = UUID.randomUUID().toString();
+    JsonArray succeedingTitles = getUnconnectedPrecedingSucceedingTitle(
+      precedingSucceedingTitleId1, precedingSucceedingTitleId2);
+    JsonObject smallAngryPlanetJson = smallAngryPlanet(UUID.randomUUID());
+    smallAngryPlanetJson.put("succeedingTitles", succeedingTitles);
+
+    IndividualResource createdInstance = instancesClient.create(smallAngryPlanetJson);
+
+    JsonArray actualSucceedingTitles = createdInstance.getJson().getJsonArray("succeedingTitles");
+    JsonObject actualSucceedingTitle1 = getRecordById(actualSucceedingTitles, precedingSucceedingTitleId1);
+    JsonObject actualSucceedingTitle2 = getRecordById(actualSucceedingTitles, precedingSucceedingTitleId2);
+
+    assertPrecedingSucceedingTitles(actualSucceedingTitle1, succeedingTitles.getJsonObject(0),
+      createdInstance.getId().toString(), null);
+    assertPrecedingSucceedingTitles(actualSucceedingTitle2, succeedingTitles.getJsonObject(1),
+      createdInstance.getId().toString(), null);
+  }
+
+  @Test
+  public void canUpdateAnInstanceWithUnconnectedPrecedingTitles()
```

Comment:
Is adding a new unconnected preceding title, removing the existing unconnected preceding title or something else?

Reply:
Yes, it is

Reply:
Sorry, I think I didn't follow. Which of those situations does this demonstrate?